docs/ops/detection/DeformablePSROIPooling_1.md

```diff
@@ -4,70 +4,77 @@
 
 **Category**: Object detection
 
-**Short description**: *DeformablePSROIPooling* computes position-sensitive pooling on regions of interest specified by input.
+**Short description**: *DeformablePSROIPooling* computes deformable position-sensitive pooling on regions of interest specified by input.
 
 **Detailed description**: [Reference](https://arxiv.org/abs/1703.06211).
 
-*DeformablePSROIPooling* operation takes two or three input tensors: with feature maps, with regions of interests (box coordinates) and an optional tensor with transformation values.
-The box coordinates are specified as five element tuples: *[batch_id, x_1, y_1, x_2, y_2]* in absolute values.
+*DeformablePSROIPooling* operation takes two or three input tensors: with position score maps, with regions of interests (ROI, box coordinates) and an optional tensor with transformation values (normalized offsets for ROI bins coordinates).
+If only two inputs are provided, position sensitive pooling with regular ROI bins position is calculated (non-deformable).
+If third input is provided, each bin position is transformed by adding corresponding offset to the bin left top corner coordinates. Third input values are usually calculated by regular position sensitive pooling layer, so non-deformable mode (DeformablePSROIPooling with two inputs).
+
+The ROI coordinates are specified as five element tuples: `[batch_id, x_1, y_1, x_2, y_2]` in absolute values.
+
 
 **Attributes**
 
 * *output_dim*
 
-  * **Description**: *output_dim* is a pooled output channel number.
+  * **Description**: *output_dim* is the number of the output channels, size of output `C` dimension.
   * **Range of values**: a positive integer
   * **Type**: `int`
   * **Default value**: None
   * **Required**: *yes*
 
-* *group_size*
-
-  * **Description**: *group_size* is the number of groups to encode position-sensitive score maps.
-  * **Range of values**: a positive integer
-  * **Type**: `int`
-  * **Default value**: 1
-  * **Required**: *no*
-
 * *spatial_scale*
 
-  * **Description**: *spatial_scale* is a multiplicative spatial scale factor to translate ROI coordinates from their input scale to the scale used when pooling.
+  * **Description**: *spatial_scale* is a multiplicative spatial scale factor to translate ROI coordinates from their input original size to the pooling input. Ratio of the input score map size to the original image. 
```

Comment:
`to the original image.` -> `to the original image size.`

Reply:
Done